nadlogar/problems/fixtures/initial.json

```diff
@@ -48,16 +48,15 @@
         }
     },
     {
-        "model": "problems.problemtext",
-        "pk": 5,
-        "fields": {
-            "content_type": [
-                "problems",
-                "razstavirazliko"
-            ],
-            "question": "Razstavi izraz ${izraz}$",
-            "answer": "${razstavljen}$"
-        }
+    "model": "problems.problemtext",
```

Comment:
Tole je vse narobe zamaknjeno. Tale zadnji commit ni niti potreben.

Reply:
Se strinjam z obojim. Če pride do konflikta z vejo `master`, je bolje, če lokalno naredite `rebase`, in nato `push --force` nazaj na vašo vejo.